app/assets/stylesheets/db/controllers/works.scss

```diff
@@ -0,0 +1,19 @@
+body.db-works, body.db-draft_works {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `db-draft_works` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line